src/threat_model/qualitative/metadata.md

```diff
@@ -0,0 +1,48 @@
+# Metadata
+
+Protection of metadata is a key goal of the Phoenix homeserver protocol. Metadata typically includes information like the sender and recipient of a message, the time it was sent, and the size of the message, and the group conversation in which it was sent. Metadata aggregation can reveal patterns of communication and can be used to infer social relationships. In particular, correlation can be used to infer social graphs.
+
+The metadata threat model considers two types of adversaries: snapshot adversaries and active observer adversaries. This section describes the two adversaries and the information they have access to.
+
+## Snapshot adversarires
+
+Snapshot adversaries have access to snapshots of the server's persisted state. They can see the metadata of all messages that have been sent and received up to the point of the snapshot and that haven't been deleted from the long-term storage. An example of a snapshot adversary is a government that has seized the server's hard drives, or a provider that is forced to comply with a government request to hand over data.
+
+The main protection against the snapshot adversary is encryption at rest for most relevant metadata. In this instance, encryption at rest means that clients hold the key material and supply it to the server for individual queries. The server can decrypt the data and process the query before re-encrypting the data and deleting the key from memory.
+
+As a consequence, snapshots become largely useless to the adversary since the senders of messages, the group in which messages were sent, etc. are encrypted. The encryption keys are either only held by clients, or are only held temporarily by the server in volatile memory and are therefore not part of the snapshot.
+
+### Data available to snapshot adversaries
+
+Despite the use of encryption at rest, the snapshot adversary can still see the following data:
+
+- For each group on the DS
+  - The rough size of the group
+  - The timestamp of the last time the group was active (e.g. due to a key update)
+- Reserved group IDs
+- AS key material
+  - Credentials of all clients (includes usernames)
+  - AS credentials and key material
+  - OPAQUE key material
+  - Privacy Pass key material
+- The number of messages in the connection queues of all clients (but not the contents of the messages)
+- All user pseudonyms and their associated client pseudonyms
+- All currently available KeyPackages of all clients
+- The number of messages in all client queues and the (padded) length of each message (but not the contents)
+- QS key material
+  - QS signing keys
+  - QS HPKE encryption keys
```

Comment:
In general, this list needs more context regarding correlation (or the lack thereof). We shouldn't assume readers understand that e.g. a client is not tied to an AS user on the server.